Sources/SCTE35/HelperConverter/BitConverter.swift

```diff
@@ -9,9 +9,11 @@
 import Foundation
 
 class BitConverter {
-    /// Will return an array of 8 bits
-    /// i.e. input = 15, output = [0, 0, 0, 0, 1, 1, 1, 1]
-    static func bits(fromByte byte: UInt8) -> [Bit] {
+    /**
```

Comment:
You'll see this change a lot.  Many code comments are one very long line.  I decided to break them up into multiple lines for readability.